scripts/hydra

```diff
@@ -355,4 +347,18 @@ function status() {
   fi
 }
 
-eval "$(argc --argc-eval "$0" "$@")"
\ No newline at end of file
+# @cmd Remote install to your nodes the dependencies to run the metagraph using Ansible
+function remote_install() {
```

Comment:
I think we need to give the option for the user to set a specific SSH key, in case they don't want to use their default. That seems like it will be a fairly common requirement.

Reply:
Also, what about `remote_configure` instead of `remote_install`? I think that's probably a more accurate description.

Reply:
I agree, let's update to remote_configure. Also, I'll add on the hosts file the option to provide the ssh key if the user wants to.